contracts/scripts/native_solc_compile_all

```diff
@@ -53,36 +53,36 @@ $SCRIPTPATH/native_solc8_6_compile ChainlinkClient.sol
 # VRF
 $SCRIPTPATH/native_solc8_6_compile vrf/VRFRequestIDBase.sol
 $SCRIPTPATH/native_solc8_6_compile vrf/VRFConsumerBase.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFConsumer.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFRequestIDBaseTestHelper.sol
+$SCRIPTPATH/native_solc8_6_compile tests/vrf/VRFConsumer.sol
+$SCRIPTPATH/native_solc8_6_compile tests/vrf/VRFRequestIDBaseTestHelper.sol
 $SCRIPTPATH/native_solc8_6_compile mocks/VRFCoordinatorMock.sol
 
 # VRF V2
 $SCRIPTPATH/native_solc8_6_compile vrf/VRFConsumerBaseV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFConsumerV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFMaliciousConsumerV2.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFTestHelper.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2RevertingExample.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2ProxyAdmin.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFV2TransparentUpgradeableProxy.sol
-$SCRIPTPATH/native_solc8_6_compile tests/VRFConsumerV2UpgradeableExample.sol
+$SCRIPTPATH/native_solc8_6_compile tests/vrf/VRFConsumerV2.sol
```

Comment:
I thought we wanted these in `/vrf/testhelpers/...` to have them live inside the vrf folder

@makramkd correct?

Reply:
Per comment `contracts/src/v0.8/tests/vrf`

Reply:
I am completely indifferent here.

Reply:
Lets name them `testhelpers` because otherwise we'd think those are actual tests.